elastic/kibana
# [maps] only show vector tile inspector panel when map uses elasticsearch vector tile API

Closes https://github.com/elastic/kibana/issues/170042

PR disables vector tile adapter when map does not contain vector tile layers

### Test
1) create new map
2) add documents layer
3) set scaling to "limit to 10000"
4) open inspector. Notice inspector only displays request adapter
5) change scaling to "vector tiles"
6) open inspector. Notice inspector displays vector tile adapter and request adapter
